src/main/java/org/gridsuite/ds/server/repository/DynamicSimulationRepository.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.ds.server.repository;
+
+import org.gridsuite.ds.server.dto.DynamicSimulationStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Repository;
+import reactor.core.publisher.Mono;
+
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+@Repository
+public class DynamicSimulationRepository {
```

Comment:
Is this class really useful?

Reply:
I like having it to do checks and error handling to keep services classes cleaner

Reply:
Ok